internal/handlers/common/update.go

```diff
@@ -902,11 +902,11 @@ func newUpdateError(code commonerrors.ErrorCode, msg, command string) error {
 // validateOperatorKeys returns error if any key contains empty path or
 // the same path prefix exists in other key or other document.
 func validateOperatorKeys(command string, docs ...*types.Document) error {
-	seen := map[string]struct{}{}
+	paths := []*types.Path{}
```

Comment:
I like the var name `seen` more because it showed what was the var about. To me `visistedPath` would be a better name because it has some semantics. But it's rather a minor comment.